skt/kerneltree.py

```diff
@@ -259,8 +259,18 @@ def __get_remote_url(self, remote):
             match = re.match('Fetch URL: (.*)', line)
             if match:
                 return match.group(1)
+        return None
```

Comment:
Same in this commit, mentioning the name of the specific warning is more useful than saying "something about the return value" :)